twitter_ads/creative.py

```diff
@@ -387,3 +387,62 @@ def preview(self):
 resource_property(ScheduledTweet, 'nullcast', transform=TRANSFORM.BOOL)
 resource_property(ScheduledTweet, 'scheduled_at', transform=TRANSFORM.TIME)
 resource_property(ScheduledTweet, 'text')
+
+
+class MediaLibrary(Resource, Persistence):
+
+    PROPERTIES = {}
+
+    RESOURCE_COLLECTION = '/' + API_VERSION + '/accounts/{account_id}/media_library'
+    RESOURCE = '/' + API_VERSION + '/accounts/{account_id}/media_library/{id}'
+
+    def reload(self, **kwargs):
+        if not self.media_key:
+            return self
+
+        resource = self.RESOURCE.format(account_id=self.account.id, id=self.media_key)
+        response = Request(self.account.client, 'get', resource, params=kwargs).perform()
+
+        return self.from_response(response.body['data'])
+
+    def save(self):
+        if self.media_key:
+            method = 'put'
+            resource = self.RESOURCE.format(account_id=self.account.id, id=self.media_key)
+        else:
+            method = 'post'
+            resource = self.RESOURCE_COLLECTION.format(account_id=self.account.id)
+
+        response = Request(
+            self.account.client, method,
+            resource, params=self.to_params()).perform()
+
+        return self.from_response(response.body['data'])
+
+    def delete(self):
+        resource = self.RESOURCE.format(account_id=self.account.id, id=self.media_key)
+        response = Request(self.account.client, 'delete', resource).perform()
+        self.from_response(response.body['data'])
+
+
+# media library properties
+# read-only
+resource_property(MediaLibrary, 'aspect_ratio', readonly=True)
+resource_property(MediaLibrary, 'created_at', readonly=True, transform=TRANSFORM.TIME)
+resource_property(MediaLibrary, 'deleted', readonly=True, transform=TRANSFORM.BOOL)
+resource_property(MediaLibrary, 'duration', readonly=True)
```

Comment:
Should this be TRANSFORM.INT?

Reply:
I'm okay with leaving it as string but either way we might consider adding a helper function to pretty print with mins and secs (or if it's easier for people to figure out how to do it with INT it should be INT)

Reply:
Yeah, makes sense. I'll update this.